integration/setup/catalog.go

```diff
@@ -0,0 +1,138 @@
+package setup
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/appcatalog"
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/afero"
+	"k8s.io/helm/pkg/helm"
+
+	"github.com/giantswarm/azure-operator/integration/env"
+	"github.com/giantswarm/azure-operator/integration/key"
+	"github.com/giantswarm/azure-operator/pkg/project"
+)
+
+const (
+	CatalogStorageURL     = "https://giantswarm.github.io/control-plane-catalog/"
+	TestCatalogStorageURL = "https://giantswarm.github.io/control-plane-test-catalog/"
+)
+
+var (
+	latestOperatorRelease string
+)
+
+func GetLatestOperatorRelease() string {
+	return latestOperatorRelease
+}
+
+func init() {
+	fmt.Printf("calculating latest %#q release\n", project.Name())
+
+	var err error
+	latestOperatorRelease, err = appcatalog.GetLatestVersion(context.Background(), CatalogStorageURL, project.Name())
+	if err != nil {
+		panic(fmt.Sprintln("cannot calculate latest operator release from app catalog"))
+	}
+
+	fmt.Printf("latest %#q release is %#q\n", project.Name(), latestOperatorRelease)
+}
+
+func pullLatestReleaseChartPackage(ctx context.Context, config Config, chartName string) (string, error) {
```

Comment:
I spend fair amount of time to decode this name. I think `pullLatestChart` would do. "Chart" is the package. "Release" is the installed chart in helm lingo.